v1/linkml-schemas/credentials.yaml

```diff
@@ -93,10 +93,24 @@ classes:
     see_also:
       - ../json-schemas/IdentityCheckCredentialJWT.json
     attributes:
+      iss:
+        range: uri
+        description: The `iss` (issuer) claim identifies the principal that issued the JWT.
+        required: true
+      sub:
+        range: uri
+        description: The `sub` (subject) claim identifies  jaja the principal that is the subject of the JWT.
+        required: true
+      nbf:
+        range: integer
+        description: The `nbf` (not before) claim identifies the time the JWT is valid from.
+        required: true
       vc:
         range: IdentityCheckCredentialClass
         required: true
-
+      jti:
+        range: uri
+        pattern: "^(?!\s*$).+"
```

Comment:
The build does not like the " character so you'll need to remove it.

Reply:
Hmm I'd have thought the quote mark here is a YAML delimiter and so never makes it through to the code, this seems weird